Makefile

```diff
@@ -525,7 +525,7 @@ docker-test-all:
 
 docker-restart-koku:
 	@if [ -n "$$($(DOCKER) ps -q -f name=koku_server)" ] ; then \
-         docker-compose restart koku-server ; \
+         docker-compose restart koku-server masu-server koku-worker koku-beat koku-listener ; \
```

Comment:
The restart of these chosen services was because the services started getting into a non-responsive state when the DB container went down and back up again. This is just a restart vs a rebuild + start. So, hopefully, it will not be so taxing.

Reply:
Yep, I understand. I'm complaining about seconds here, and just being impatient. I'll make the `lite` version of the command next time I need it. I have a suspicion that the non-responsive state is from saving a file while the DB is down, which triggers a server reload for koku/masu and celery to pick up changes as well.